backend/seeds.py

```diff
@@ -35,6 +62,24 @@ def seed_db_command():
             },
             "admin",
         )
+
+    for volunteer in volunteers:
+        volunteer = register_volunteer(
+            {
+                "first_name": volunteer.first_name,
+                "last_name": volunteer.last_name,
+                "email": f"{volunteer.last_name.lower()}@example.com",
+                "role": volunteer.role,
+                "phone": volunteer.phone,
+                "zone": volunteer.zone,
+                "address": volunteer.address,
+                "zone_address": volunteer.zone_address,
```

Comment:
Change this field as not require and this row can be removed

Reply:
zone_address changed to not required